portfolio/src/main/webapp/script.js

```diff
@@ -190,8 +190,45 @@ function createNextButton(direction, isValid) {
   return nextButton;
 }
 
+/**
+ * Retrieves markers from /markers.
+ */
+function fetchMarkers() {
+  fetch('/markers').then(response => response.json()).then((markers) => {
+    markers.forEach(
+        (marker) => {
+            createGameMarker(marker.lat, marker.lng)});
+  });
+}
+
+/**
+ * Creates a marker on the interactive map.
+ */
+function createGameMarker(lat, lng) {
+  return new google.maps.Marker({position: {lat: lat, lng: lng}});
```

Comment:
nit: I think this can just be `return new google.maps.Marker({position: {lat, lng}});`